windows/ReactTestApp/ReactTestApp.vcxproj

```diff
@@ -70,19 +68,24 @@
         <WholeProgramOptimization>true</WholeProgramOptimization>
         <LinkIncremental>false</LinkIncremental>
     </PropertyGroup>
+    <PropertyGroup Label="UserMacros">
+        <BundleContentRoot>..\..\example\dist</BundleContentRoot>
+        <ManifestRootPath>..\..\example\</ManifestRootPath>
+        <BundleContent>$(BundleContentRoot)\**\*</BundleContent>
+        <ReactNativeModulePath>..\..\node_modules\react-native-windows\</ReactNativeModulePath>
+    </PropertyGroup>
```

Comment:
This is needed if we want to run app directly from react-native-test-app/windows. If we will only run generated application, then this can be removed. Furthermore, we can then simplify solution file as well, so they will become more like templates for generating apps.

Reply:
For all the other platforms, we are developing them through the Example app. If we are able to do that now, I would remove this block and simplify things as you've said.

Reply:
I think it should be possible if we reference source files, like you said [here](https://github.com/microsoft/react-native-test-app/pull/192#discussion_r469099065).